e2e/transactions/transactions_test.go

```diff
@@ -229,6 +229,8 @@ func (s *TransactionsTestSuite) TestSendEther() {
 	s.StartTestBackend(params.StatusChainNetworkID)
 	defer s.StopTestBackend()
 
+	s.NoError(EnsureNodeSync(s.Backend.NodeManager()))
```

Comment:
Just a nitpick, but for consistency with other code, it's better to split this into two lines.

Reply:
Where is it 2 lines? It's one line everywhere, as I can see.
Also it will be every shorter soon as we'll be throwing panic there: https://github.com/status-im/status-go/pull/410#issuecomment-338487751

Reply:
@tiabc @themue I mean we usually use this form:

```go
   err := Foo()
   require.NoError(err)
```
not a big deal, of course, but that catched my attention, 

Anyway, that was a nitpick, so it should not be discussed longer than 5 seconds :)